python/ray/tune/integration/pytorch_lightning.py

```diff
@@ -162,7 +162,7 @@ class TuneReportCallback(TuneCallback):
     """
 
     def __init__(self,
-                 metrics: Union[str, List[str], Dict[str, str]],
+                 metrics: Union[None, str, List[str], Dict[str, str]] = None,
```

Comment:
It's been a really long time since I've used python type checking but shouldn't we do this?
```suggestion
                 metrics: Optional[Union[str, List[str], Dict[str, str]]] = None,
```
I'm pretty sure they're exactly the same, but Optional seems nicer

Reply:
Hm, so the nice thing about keeping it the way it is is that it is a bit easier to read (because we don't introduce another bracket). Also, in this particular case we would have to line-break, as otherwise the linter will complain. So for readability, in this case, I'd like to keep it as is.